sympy/physics/mechanics/system.py

```diff
@@ -159,13 +161,12 @@ class System(_Methods):
     [bob_mass*l*u_pin**2*sin(q_pin) + F],
     [          -bob_mass*g*l*sin(q_pin)]])
 
-    The complexity of the above example can be increased if we add a holonomic
-    constraint, to prevent the particle from moving in the horizontal (x)
-    direction. This can be done by adding a holonomic constraint. After, which
-    we should also redefine what our (in)dependent generalized coordinates and
-    speeds are. Note that the bakcend for forming the equations of motion is
-    resetted automatically, because the system properties are changed in this
-    process.
+    The complexity of the above example can be increased if we add a constraint
+    to prevent the particle from moving in the horizontal (x) direction. This
+    can be done by adding a holonomic constraint. After which we should also
+    redefine what our (in)dependent generalized coordinates and speeds are. Note
+    that the bakcend for forming the equations of motion is reset automatically
+    because the system properties are changed in this process.
```

Comment:
```suggestion
    The complexity of the above example can be increased if we add a constraint
    to prevent the particle from moving in the horizontal (x) direction. This
    can be done by adding a holonomic constraint. After which we should also
    redefine what our (in)dependent generalized coordinates and speeds are. Note
    that the backend for forming the equations of motion is reset automatically
    because the system properties are changed in this process.
```